src/jobsautoreport/report.py

```diff
@@ -32,14 +35,14 @@ class Report(BaseModel):
     number_of_successful_e2e_or_subsystem_periodic_jobs: int
     number_of_failing_e2e_or_subsystem_periodic_jobs: int
     success_rate_for_e2e_or_subsystem_periodic_jobs: float
-    top_10_failing_e2e_or_subsystem_periodic_jobs: list[tuple[str, JobStatesCount]]
+    top_10_failing_e2e_or_subsystem_periodic_jobs: list[tuple[str, JobMetrics]]
     number_of_e2e_or_subsystem_presubmit_jobs: int
     number_of_successful_e2e_or_subsystem_presubmit_jobs: int
     number_of_failing_e2e_or_subsystem_presubmit_jobs: int
     number_of_rehearsal_jobs: int
     success_rate_for_e2e_or_subsystem_presubmit_jobs: float
-    top_10_failing_e2e_or_subsystem_presubmit_jobs: list[tuple[str, JobStatesCount]]
-    top_5_most_triggered_e2e_or_subsystem_jobs: list[tuple[str, int]]
+    top_10_failing_e2e_or_subsystem_presubmit_jobs: list[tuple[str, JobMetrics]]
```

Comment:
why not include the job name in job metrics?

Reply:
Because presubmit/periodic success_rate doesn't use job_name, but uses this object. I will create another object for that